docs/releases/1.10.txt

```diff
@@ -35,6 +35,8 @@ Minor features
 * For sites running on a subpath, the default :attr:`URL for the "View site"
   link <django.contrib.admin.AdminSite.site_url>` at the top of each admin page
   will now point to ``request.META['SCRIPT_NAME']`` if set, instead of ``/``.
+* :mod:`admin_static <django.contrib.admin.templatetags.admin_static` is now
```

Comment:
From jenkins:
/home/jenkins/workspace/docs/docs/releases/1.10.txt:38: WARNING: py:mod reference target not found: admin_static <django.contrib.admin.templatetags.admin_static

Missing a closing parens? (or maybe it's complaining that it's undocumented?)